src/tabs/chat/ChatTabModel.tsx

```diff
@@ -0,0 +1,70 @@
+import { IncomingMessage } from "../../io/IncomingMessage";
+import { ResourcePath } from "../../io/ResourcePath";
+import { ResourcePathUtil } from "../../io/ResourcePath";
+import { ResourceSubscriberTabModel } from "../ResourceSubscriberTabModel";
+import { TabController } from "../TabController";
+import { ChatTabController } from "./ChatTabController";
+import { Message, ChatTabState } from "./ChatTabState";
+import { OnlinePlayerMetadata } from "../../io/OnlinePlayerMetadata";
+import { ActionResult } from "../../io/ActionResult";
+
+export class ChatTabModel extends ResourceSubscriberTabModel<ChatTabState> {
+
+  public getName(): string {
+    return "Chat";
+  }
+
+  public getSubscribedResourcePaths(): ResourcePath[] {
+    return [
+      ["console", "onlinePlayers"],
```

Comment:
You don't need to subscribe to the console resource. The subscribing is only done if you need the GET resource, which in this case is the command list.